theme/static/js/hs-vue/subject-keywords-app.js

```diff
@@ -31,16 +34,19 @@ let subjKeywordsApp = new Vue({
 
             if (this.newKeyword.trim() === "") {
                 return; // Empty string detected
+            }else if (this.newKeyword.length > 500) {
```

Comment:
There is a limit of 100 characters in the backend for keyword. 
https://github.com/hydroshare/hydroshare/blob/develop/hs_core/models.py#L1762

Reply:
@pkdash thanks for checking! Are you sure that is the right model for the Subject Keywords? Because 100+ char keywords seem to persist in the db from what I can see:

https://user-images.githubusercontent.com/17934193/157127198-32a4f8fa-01f6-4d2b-9eda-9cd4c6bd87c2.mov

Reply:
I tested in prod and it only persists the first 100 characters in db. You should be able to test it using your branch. Once you add a keyword with more than 100 chars, go to resource view mode (which does a page reload) and you will see it only stores the first 100 chars.

Reply:
100% correct thanks @pkdash, I shouldn't have doubted you! :)
I've revised from 500 to 100